### PR TITLE
{chem}[GCC/8.3.0,iccifort/2019.5.281] IGMPlot v2.4.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2610.swalot.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c2ab0869b384c5af66f8e808ec8e680f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3109.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/0bb9e3623fa747b7fd24c530f54d65c9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/aa74fbd52a477f8e0979f0ed7baff88f for a full test report.

File: easybuild/easyconfigs/i/IGMPlot/IGMPlot-2.4.2-GCC-8.3.0-2.32.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+easyblock = 'MakeCp'
+
+name = 'IGMPlot'
+version = '2.4.2'
+
+homepage = 'http://igmplot.univ-reims.fr'
+description = """IGMPlot is a free open-source program developed to identify molecular interactions and
+ prepare data to build 2D and 3D representations of them in the molecular environment."""
+
+toolchain = {'name': 'GCC', 'version': '8.3.0-2.32'}
```
Comment (akesandgren) [inline easybuild/easyconfigs/i/IGMPlot/IGMPlot-2.4.2-GCC-8.3.0-2.32.eb]: That should be just 8.3.0 now right?

Comment (boegel) [reply easybuild/easyconfigs/i/IGMPlot/IGMPlot-2.4.2-GCC-8.3.0-2.32.eb]: Yeah, good point...

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/fef7f2d8345ad75b2fe6a2ac36c5fa4b for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**